src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1413,9 +1412,32 @@ pre.rust {
 		transform: rotate(360deg);
 	}
 }
-#settings-menu.rotate img {
+#settings-menu.rotate > a img {
 	animation: rotating 2s linear infinite;
 }
+#settings-menu #settings {
+	position: absolute;
+	right: 0;
+	z-index: 1;
+	display: block;
+	margin-top: 7px;
+	border-radius: 3px;
+	border: 1px solid;
+}
+#settings-menu #settings .setting-line {
+	margin: 0.6em;
+}
+#settings-menu #settings::before {
```

Comment:
Please add a comment indicating that this draws the little arrow connecting the settings menu to the gear icon.